src/main/java/step3/RandomRule.java

```diff
@@ -0,0 +1,20 @@
+package step3;
+
+import java.util.Random;
+
+public class RandomRule implements Rule {
+
+    private static final int MAX_NUMBER = 10;
+    private static final int CONDITION_NUMBER = 4;
+
+
+    private static final Random random = new Random();
+
+    @Override
+    public boolean apply() {
+        if (random.nextInt(MAX_NUMBER) >= CONDITION_NUMBER) {
+            return true;
+        }
+        return false;
```

Comment:
already return을 쓰면 어떨까요?
```suggestion
return random.nextInt(MAX_NUMBER) >= CONDITION_NUMBER
```